DahlDesignDDC/20_Setup.ino

```diff
@@ -115,5 +115,10 @@ void setup()
 {
   setupLeds();
   LEDStartup();
+}
+#elif
```

Comment:
Might need to add `(BOARDTYPE == 2)` on this `#elif`, or we might have issues with other boards?